openapi-manager/README.md

```diff
@@ -0,0 +1,296 @@
+# Dropshot OpenAPI manager
+
+This crate provides most of the scaffolding to manage OpenAPI documents generated by [Dropshot](https://docs.rs/dropshot), using its support for **API traits**.
```

Comment:
I've tried making this generic and not Omicron-specific, but please ensure that as part of the review.